build_all.sh

```diff
@@ -20,12 +20,16 @@ if [ $? != 0 ] ; then usage ; fi
 
 eval set -- "$OPTS"
 
-JOBS="`cat /proc/cpuinfo | grep processor | wc -l`"
+# JOBS="`cat /proc/cpuinfo | grep processor | wc -l`"
+JOBS=4  # Core count
+
 
 MAKE_Z3_AT=""
 MAKE_STAREXEC=""
 MAKE_DEBUG=""
 
+eval $(opam env)
+
```

Comment:
Let's not have opam initialization within our script. We can assume the environment to be properly setup.

Reply:
Oops sorry.